docs/csharp/whats-new/csharp-7.md

```diff
@@ -521,8 +520,8 @@ feature:
 [!code-csharp[UsingValueTask](../../../samples/snippets/csharp/new-in-7/AsyncWork.cs#30_UsingValueTask "Using ValueTask")]
 
 > [!NOTE]
-> You need to add the pre-release NuGet package `System.Threading.Tasks.Extensions`
-> in order to use `ValueTask` in Visual Studio 15 Preview 5.
+> You need to add the NuGet package `System.Threading.Tasks.Extensions`
```

Comment:
same here ... I'd suggest just for safety against a crazy linefeed to put this on one line. It might not make a difference either way, but who knows. The :pray: server gods :pray: may smile on us if the note is on one line.

> You need to add the NuGet package `System.Threading.Tasks.Extensions` in order to use `ValueTask`.